.gitignore

```diff
@@ -47,15 +47,21 @@
 ehthumbs.db
 Thumbs.db
 
-# build directory #
-###################
+# Build directories (Gradle)
 classes/
 build/
 
-# tooling and project files #
-#############################
+# Build directories (IntelliJ)
+out/
+
+# Tooling and project files
 prj/
 .idea/
 .gradle/
 !gradle-wrapper.jar
-TAGS
```

Comment:
Looks like you didn't merge the previous change (Ignore TAGS)

Reply:
Done.